api.js

```diff
@@ -1,10 +1,32 @@
-const exec = require("child_process").exec;
+const { exec, spawn } = require("child_process");
 const fs = require("fs");
+const kill = require('tree-kill');
 
 const processes = {};
 
 const options = { maxBuffer: 1024 * 1000000 };
 
+const startServerScript = (script) => {
+  try {
+    const commandProcess = spawn('npm', ['run', script]);
```

Comment:
I think we should also pass in `--silent` just like I do on line 65 to skip all the typical npm logs that are not useful